simple-singolo/scss/_footer.scss

```diff
@@ -0,0 +1,86 @@
+footer {
+  background-color: $main-color;
+  border-top: 6px solid $header-border;
+  width: 100%;
+  margin: 0;
+}
+
+.social_media_container {
+  display: flex;
+  flex-direction: row;
+  padding: 22px 42px;
+}
+
+.footer {
+  @include header-footer-flex;
+
+  @media screen and (max-width: $tablet) {
+    justify-content: center;
+  }
+}
+
+.copyright {
+  color: $portfolio-text;
+  padding: 30px 0 29px 40px;
+  font-size: 1.3rem;
+
+  @media screen and (max-width: $tablet) {
+    display: none;
+  }
+}
+
+.facebook_link {
+  @include social-media(-250px, -169px);
+
+  margin-left: 8px;
+
+  &:hover {
+    @include social-media(-166px, -170px);
+  }
+
+  &:active {
+    @include social-media(-166px, -223px);
+  }
+}
+
+.google_link {
+  @include social-media(-250px, -116px);
+
+  margin-left: 8px;
+
+  &:hover {
+    @include social-media(-114px, -170px);
+  }
+
+  &:active {
+    @include social-media(-114px, -223px);
+  }
+}
+
+.twitter_link {
+  @include social-media(-250px, -63px);
+
+  margin-left: 8px;
+
+  &:hover {
+    @include social-media(-62px, -170px);
+  }
+
+  &:active {
+    @include social-media(-62px, -223px);
+  }
+}
+
+.linkedin_link {
+  @include social-media(-250px, -10px);
+
+  margin-left: 8px;
```

Comment:
Add one more cll to all those links in markup (e.g. `social-icon`). By doing this you'll be able to set `social-icon { margin-left: 8px }`  without duplicating same style many times.

Reply:
done, thanks